lightning/03 Configuring and Using CTI Adapter Features/01 CTI Adapter Configuration.md

```diff
@@ -51,6 +51,13 @@ and supervisors that will be using the platform.
     the presence rules to sync state from Amazon Connect to Salesforce
     Omni-Channel.
 
+10. **Audio Device Settings** Turning this setting on allows the Agent
```

Comment:
Same as above, doesn't have my newest changes, which include the troubleshooting step